### PR TITLE
Change some alerts on improve translations to helpbox

### LINKED ISSUE #22193: [IMPROVEMENT] Replace info alerts by a more discreet alternative
#### Is your feature request related to a problem?

These info alerts seem to be too important for this page and need to be discreet because users don't want to read them every time they're on this translation page.

Frame 194

#### Describe the solution you'd like

- Solution 1: Change these info alerts' behaviours so they can be closed. 
- Solution 2: Remove these info alerts and replace them by tooltips placed next to the card title. This solution can be applied when the info alerts is about the full card.   

Frame 197

ping @NeOMakinG

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Alert infos are not the component that should be used sometimes, it needs to be replaced into helpbox in the header of cards
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #22193.
| How to test?  | See issue

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22373)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/administration.html.twig [hunk 1]
```diff
@@ -133,16 +133,20 @@
...
             <i class="material-icons">priority_high</i> {{ 'Notifications'|trans }}
+
+            <span
+              class="help-box"
+              data-container="body"
+              data-toggle="popover"
+              data-trigger="hover"
+              data-placement="right"
+              data-content="{{ "Notifications are numbered bubbles displayed at the very top of your back office, right next to the shop's name. They display the number of new items since you last clicked on them."|trans({}, 'Admin.Advparameters.Help') }}"
+              title
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/administration.html.twig]:
```suggestion
              title=""
```

File: src/PrestaShopBundle/Resources/views/Admin/Improve/International/Geolocation/Blocks/geolocation_ip_address_whitelist.html.twig [hunk 2]
```diff
@@ -30,18 +30,20 @@
...
       {{ 'IP address whitelist'|trans }}
+
+      <span
+        class="help-box"
+        data-container="body"
+        data-toggle="popover"
+        data-trigger="hover"
+        data-placement="right"
+        data-content="{{ 'You can add IP addresses that will always be allowed to access your shop (e.g. Google bots\' IP).'|trans({}, 'Admin.International.Help') }}"
+        title
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/International/Geolocation/Blocks/geolocation_ip_address_whitelist.html.twig]:
```suggestion
        title=""
```

File: src/PrestaShopBundle/Resources/views/Admin/Improve/International/Geolocation/Blocks/geolocation_options.html.twig [hunk 3]
```diff
@@ -30,18 +30,20 @@
...
       {{ 'Options'|trans({}, 'Admin.Global') }}
+
+      <span
+        class="help-box"
+        data-container="body"
+        data-toggle="popover"
+        data-trigger="hover"
+        data-placement="right"
+        data-content="{{ 'The following features are only available if you enable the Geolocation by IP address feature.'|trans }}"
+        title
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/International/Geolocation/Blocks/geolocation_options.html.twig]:
```suggestion
        title=""
```

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig [hunk 4]
```diff
@@ -31,23 +31,20 @@
...
         <i class="material-icons">filter_none</i> {{ 'Handling'|trans }}
+
+        <span
+          class="help-box"
+          data-container="body"
+          data-toggle="popover"
+          data-trigger="hover"
+          data-placement="right"
+          data-content="{{ 'If you set these parameters to 0, they will be disabled.'|trans({}, 'Admin.Shipping.Help') }} {{ 'Coupons are not taken into account when calculating free shipping.'|trans({}, 'Admin.Shipping.Help') }}"
+          title
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig]:
```suggestion
          title=""
```

Comment (NeOMakinG) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig]: Useless to me, but why not if it seems clearer to you

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
The issue #22193  is waiting for @PrestaShop/product-team.

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Roomsmaniia** [comment]
LGTM,

Thanks @NeOMakinG

**Comment by atomiix** [comment]
Thanks @NeOMakinG & @RomainBocheux

**Pull request merged**